cli/prisma2/src/bin.ts

```diff
@@ -91,7 +99,20 @@ async function main(): Promise<number> {
     console.error(result)
     return 1
   }
-  console.log(result)
+
+  const checkResult = await checkpoint.check({
+    product: 'prisma',
+    version: packageJson.version,
+    cache_only: true,
```

Comment:
What kind of cache is this(I am assuming checkpoint server request cache?) ? Not very clear, also docs of the client doesn't mention this option: https://www.npmjs.com/package/checkpoint-client

I'd assume this option will enable you to not send the request to the server at all on a first look.

Reply:
Good call. I'll update the docs. I'm not very happy with the name of this property. The intent is to not go to the network at all, only check the local client-side cache. If it's there, great, return it. Otherwise, return undefined.

Reply:
Name is fine in that case. I guessed it correctly. Just update the docs :)

Reply:
Some suggestions: `localOnly`, `noNetwork`, `onlyCache`

Reply:
simplified the package to be able to remove this entirely.